tests/unit/Controller/ConfigurationControllerTest.php

```diff
@@ -128,7 +128,7 @@ public function testCreateWithCopy() {
 							break;
 						default: $expectedValue = null;
 					};
-					return $this->stringStartsWith('tgt');
```

Comment:
`stringStartsWith` returns an object that is a "constraint" object within `phpunit` `Assert`. 
Actually here we are supposed to return a boolean that indicates if `$key` starts with `tgt`